code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -205,6 +205,7 @@
 		/obj/item/storage/box/m94 = 30,
 		/obj/item/storage/box/m94/cas = 10,
 		/obj/item/storage/box/recoilless_system = 1,
+		/obj/item/storage/box/t26_system = 1,
```

Comment:
```suggestion
		/obj/item/storage/box/t26_system = 1,
```
put these in the operational supply locker so there's a better tight grasp on the amount of these per round, should really go with all spec guns or whatever power crate garbage we make
I'll do it one day when i'm not tired and lazy

Reply:
> put these in the operational supply locker so there's a better tight grasp on the amount of these per round, should really go with all spec guns or whatever power crate garbage we make
> I'll do it one day when i'm not tired and lazy

I intend to change the amount of vendors on both maps.
As of now, the PoS will have 3 and Sulaco will have 2 with the current iteration.

Basically you get the same amount of SGs as the number of RRs on a ship.

Reply:
this is dumb and stupid jsut put them in the operational supply vendor so its more finely tuned and less obfsucated codewise
I'll do that after this pr